framework/include/cppmicroservices/Constants.h

```diff
@@ -317,6 +317,12 @@ namespace cppmicroservices
         US_Framework_EXPORT extern const std::string
             FRAMEWORK_BUNDLE_VALIDATION_FUNC; // = "org.cppmicroservices.framework.bundle.validation.function"
 
+        /**
+         * Framework bundle validation property specifying a all bundles validated by this framework
+         */
+        US_Framework_EXPORT extern const std::string
```

Comment:
I understand the purpose of this, to allow DS to see state from the core framework, but I really would like to see a different, less public facing mechanism used. The use of a customer facing set of properties should be avoided. Is there any other way to solve this issue? Thanks.